chapters/methodology.tex

```diff
@@ -62,6 +62,23 @@ \subsection{Architecture}
 allows easy databinding to the UI (view), whilst allowing easy access from the
 back end (model). \cite{mvvm}
 
+\subsection{Data Collection} \label{sec:data_collection}
+
+As shown in \ref{fig:arch1}, the image analyser produces \texttt{DetectionResult}s.
+In order to collect multiple \texttt{DetectionResult}s, a
+class called \texttt{Measurement} will collect multiple detection results as
+well as the results of the questionnaire.
+After an successful or unsuccessful run of the application, the result is  
+stored into an local database managed by a database manager.

```

Comment:
I suggest using `Results of a finished run (successful/unsuccessful) will be stored in a local database.`